Dockerfile

```diff
@@ -1,17 +1,25 @@
 # SPDX-License-Identifier: Apache-2.0
 
-FROM ubuntu:24.04 AS base
+FROM ubuntu:20.04 AS base
 
 RUN useradd --system --user-group --home-dir /home/autodba --shell /bin/bash autodba
 
-RUN apt-get update
-RUN apt-get install -y --no-install-recommends \
+RUN apt-get update && apt-get install -y --no-install-recommends \
     curl            \
     jq              \
-    nodejs          \
-    npm             \
     procps          \
-    wget
+    wget            \
+    gnupg           \
```

Comment:
This does not seem to be needed. I remove it and everything builds successfully. Can you please check again yourself, too?